messagix/client.go

```diff
@@ -393,3 +395,7 @@ func (c *Client) GetTaskId() int {
 	c.activeTasks = append(c.activeTasks, id)
 	return id
 }
+
+func (c *Client) Context() context.Context {
+	return c.ctx
+}
```

Comment:
This is probably not necessary, I don't think anything outside the package should use the context

```suggestion
```